src/NeoSharp.Core/Cryptography/ECPoint.cs

```diff
@@ -51,10 +51,22 @@ private ECPoint()
         /// <param name="point">Point</param>
         public ECPoint(byte[] point)
         {
-            IsInfinity = false;
+            if (point == null)
```

Comment:
or point.Length==0?

Reply:
yahh 
The issue that I had was only because this was null.
I will had the length verification too.